### ISSUE TITLE
bug/ModuleNotFoundError: No module named 'docx.text.hyperlink'

### ISSUE THREAD

**Issue by ron-unstructured** [Unstructured-IO/unstructured#1776]
**Describe the bug**
running unstructured-ingest generates the `ModuleNotFoundError`

**To Reproduce**
unstructured-ingest \
  local \
  --input-path example-docs \  
  --output-dir local-ingest-output \
  --num-processes 2 \
  --recursive \
  --verbose

**Expected behavior**
It should generate JSON file

**Screenshots**
<img width="786" alt="Screenshot 2023-10-17 at 10 32 37 AM" src="https://github.com/Unstructured-IO/unstructured/assets/138828701/9bce4541-6a40-4f10-b897-3236193b0ea2">

**Environment Info**
OS version:  macOS-13.5.2-arm64-arm-64bit
Python version:  3.10.12
unstructured version:  0.10.24
unstructured-inference version:  0.6.6
pytesseract version:  0.3.10
Torch version:  2.0.1
Detectron2 is not installed
PaddleOCR is not installed
Libmagic version:  ==> libmagic: stable 5.45 (bottled)
LibreOffice version:  ==> libreoffice: 7.6.2

**Comment by cragwolfe** [comment]
CC @scanny . perhaps an old version of python-docx installed in this case?

**Comment by scanny** [comment]
Yep, definitely. The `docx.text.hyperlink` module was added in `v1.0.0` although dependency is currently `python-docx>=v1.0.1`.

**Comment by scanny** [comment]
Also `pip install -U python-docx` if a quick-fix is good enough :)
